_posts/en/newsletters/2019-05-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+---
+title: 'Bitcoin Optech Newsletter #47'
+permalink: /en/newsletters/2019/05/21/
+name: 2019-05-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the bip-anyprevout soft fork proposal, summarizes
+a few technical talks from the Magical Crypto Friends conference, and
+includes our regular sections on bech32 sending support and notable
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed anyprevout sighash modes:** two weeks ago, Anthony Towns
```

Comment:
AJ's mailing list post (https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-May/016929.html) adds some nice context to the proposed BIP, so I think it'd be nice to link to it from here.